bitnami/memcached/templates/hpa.yaml

```
@@ -0,0 +1,34 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "memcached.fullname" . }}
```

Use this macro instead:
```suggestion
  name: {{ template "common.names.fullname" . }}
```